### PR TITLE
Update app-configuration-managed-home-screen-app.md

### PR DESCRIPTION
**Pull request by chmaguir**
update to doc

### CODE REVIEW

**Comment by PRMerger17** [comment]
@chmaguir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/app-configuration-managed-home-screen-app.md [hunk 1]
```diff
@@ -89,7 +89,9 @@ The following table lists the Managed Home Screen available configuration keys,
...
 | Show flashlight setting | bool | FALSE | Turning this setting to True allows the   end user to on or off the device's flashlight. If the device doesn't support   a flashlight, then this setting won't appear, even if configured   to True. | ✔️ |
-| Show device info setting | bool | FALSE | True allows end users to access quick info about the device from the Managed Setting app   or swipe-down. Accessible information includes device's make, model, and   serial number. | ✔️ |
+| Show device info setting | bool | FALSE | True allows end users to access quick info about the device from the Managed Setting app   or swipe-down. Accessible information includes device's make, model, and serial number for OS 8. | ✔️ |
+| Show device's name on MHS | bool | FALSE | Turn this setting to True to easily view the device's Intune portal "device name" property from the Managed Settings app or from swipe-down when **Show device info setting** is set to true. Make sure to also include the string property "Device's name," which is auto-populated by Intune with the correct value. | ❌ |
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-managed-home-screen-app.md:93]:
```suggestion
| Show device's name on MHS | bool | FALSE | Turn this setting to True to easily view the device's Intune portal "device name" property from the Managed Settings app or from swipe-down when **Show device info setting** is set to True. Make sure to also include the string property "Device's name," which is auto-populated by Intune with the correct value. | ❌ |
```

File: memdocs/intune/apps/app-configuration-managed-home-screen-app.md [hunk 2]
```diff
@@ -89,7 +89,9 @@ The following table lists the Managed Home Screen available configuration keys,
...
+| Show serial number for all supported OS version on MHS | choice | {{serialnumber}} | Ensure that in app config device_serial_number is configured to display {{SerialNumber}} **Show device info setting** is set to true. This value is auto populated by Intune with the correct value. | ❌ |
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-configuration-managed-home-screen-app.md:94]:
```suggestion
| Show serial number for all supported OS version on MHS | choice | {{serialnumber}} | Ensure that in-app config device_serial_number is configured to display {{SerialNumber}} and **Show device info setting** is set to True. This value is auto-populated by Intune with the correct value. | ❌ |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chmaguir, kindly commit the minor suggested changes. Many thanks.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@chmaguir Please implement the suggested changes. Thanks

**Comment by PRMerger19** [comment]
@Erikre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by PRMerger4** [comment]
@tiburd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**